src/microbe_stage/MicrobeStage.cs

```diff
@@ -539,6 +554,23 @@ public void ToggleGUI()
         hudRoot.Visible = !hudRoot.Visible;
     }
 
+    /// <summary>
+    ///   Called when the player died out in a patch and selected a new one
+    /// </summary>
+    public void GoToNewPatch(Patch patch)
+    {
+        if (CurrentGame == null)
+            throw new InvalidOperationException("Going to a new patch without a game setup");
+
+        CurrentGame.GameWorld.Map.CurrentPatch = patch;
+        UpdatePatchSettings();
+        playerExtinctInCurrentPatch = false;
+        SpawnPlayer();
+
+        // Auto save is wanted once possible
+        wantsToSave = true;
```

Comment:
I don't think saving is wanted in this case at all... seems like a bad idea to delete recent autosaves where the player might better be able to recover...